pkg/crc/output/output.go

```diff
@@ -4,6 +4,6 @@ import (
 	"fmt"
 )
 
-func Out(s string, args ... interface{}) {
+func Out(s string, args ...interface{}) {
 	fmt.Println(fmt.Sprintf(s, args...))
```

Comment:
@gbraad Not related to this PR but, we should also have something which doesn't insert a newline, and just `fmt.Print`

Reply:
@gbraad avoid using `Sprintf` if you need it for formatting, directly use `Printf` which should work as expected.

Reply:
@anjannath I only write code that I need... as code has to evolve instead of strictly written out for the sake of having all options. I saw no direct need for the non-newline option. Yes, I thought of it, but also remember the horrible output we got with writing intermediate steps which never aligned correctly. Until we really need...